realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -142,6 +148,50 @@
     }
 
     /**
+     * Initializes the Realm library. It is required to call this method before interacting with any other of the Realm
```

Comment:
I think we should explain what we mean by "initialize". Besides loading `.so` files (not important for users of Realm), a default configuration is created and a logger is set up.